spec/requests/users_spec.rb

```diff
@@ -0,0 +1,18 @@
+# spec/requests/users_spec.rb
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /users' do
+    it 'renders the index template' do
+      get users_path
+      expect(response).to render_template(:index)
+    end
+
+    it 'responds with a 200 status code' do
+      get users_path
+      expect(response).to have_http_status(200)
+    end
```

Comment:
- [ ] Great work writing these tests. For the `index` action of the `Users` controller, kindly write a test that ensures the response body includes the correct placeholder text. 💻 🖱️ 

Example : `expect(response.body).to include("Welcome to user index page")`

#### N.B: Please ensure that all tests pass when you are done. 🔑